apps/src/code-studio/header.js

```diff
@@ -93,7 +93,7 @@ header.build = function (scriptData, stageData, progressData, currentLevelId, pu
     $('.header_popup_link_glyph').html('&#x25B2;');
     $('.header_popup_link_text').text(dashboard.i18n.t('less'));
     $(document).on('click', hideHeaderPopup);
-    lazyLoadPopup();
+    progress.renderMiniView($('.user-stats-block')[0], scriptName, currentLevelId);
```

Comment:
I considered that, but given how heavily jquery is currently used in this file, I thought it would feel weirder to suddenly switch to the non-jquery version here.